### ISSUE TITLE
EXAMPLE 3

### ISSUE THREAD

**Issue by PowerShellToday** [MicrosoftDocs/office-docs-powershell#1096]
Seems like the verb Get implies that you don't Set?
```
Get-SPOSite -Identity https://contoso.sharepoint.com -DisableSharingForNonOwnersStatus
```
Example 3 disables the ability for non owners of a site collection to share.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2fbda735-74ec-2f4e-9c75-dd133e87320e
* Version Independent ID: 599c8c2e-6e02-f27f-1009-2640296d5d10
* Content: [Get-SPOSite (sharepoint-online)](https://docs.microsoft.com/en-us/powershell/module/sharepoint-online/Get-SPOSite?view=sharepoint-ps)
* Content Source: [sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/master/sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md)
* Service: **sharepoint-powershell**
* Product: **unspecified**
* GitHub Login: @techwriter40
* Microsoft Alias: **kirks**

**Comment by AndreaBarr** [comment]
@PowerShellToday Thank you for this feedback

I want to make sure this gets addressed properly so I am sending it over to the PowerShell Team. 

I was able to get this into the backlog for the PowerShell team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated. 
 
I made a note to request the team to update this when the work is complete.
